Kajabi/sage-lib
# [SAGE-289] Component Loader Next update

## Description

Update Loader component with new designs.

### Changes

- update svg's to align with the updated visuals
- update Rails helper to link to correct Figma design
- Include new `text` prop in rails and react and align rails documentation for props

## Designs
[Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/Sage?node-id=3452%3A25585)


## Screenshots

|  Before  |  After  |
|--------|--------|
| ![before-loader](https://user-images.githubusercontent.com/24237393/164047754-9e2d2280-6d9a-4ccd-ad16-21c39b3218f0.png) | ![after-loader](https://user-images.githubusercontent.com/24237393/164047780-85262686-fe2d-4c14-8a70-0bb400bc1adb.png) |

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/loader?sage_theme=sage_theme_next
2. Verify that the component matches [updated designs](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/Sage?node-id=3452%3A25585)

## Related
[SAGE-289](https://kajabi.atlassian.net/browse/SAGE-289)
